manifests/rules/out/pop3.pp

```diff
@@ -0,0 +1,7 @@
+# manage out pop3
+class nftables::rules::out::pop3 {
+  nftables::rule {
+    'default_out-pop3':
+      content => 'tcp dport {145, 995} accept',
```

Comment:
Shouldn't this be `110` instead of `145`?

```
$ grep -i pop3 /etc/services 
pop3		110/tcp		pop-3		# POP version 3
pop3s		995/tcp				# POP-3 over SSL
````

Reply:
Doh, good catch! I had mixed it up with imap :/